test/integration/ipamperf/test-performance.sh

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env bash
+
```

Comment:
Do we really need this whole script? Can't we re-use the test-performance.sh for scheduler-perf somehow>

Reply:
The test-performance itself was mostly reusing the code in the shared libs, as does this one for starting, stopping etcd etc.,. Most of the code is in the test specific flags setup.